lib/prometheus_exporter/server.rb

```diff
@@ -16,3 +16,4 @@
 require_relative "server/unicorn_collector"
 require_relative "server/active_record_collector"
 require_relative "server/shoryuken_collector"
+require_relative "server/resque_collector"
```

Comment:
we should do a separate PR, but I wonder now that we have so many of these if we should just switch to autoloading so we don't load up all this code upfront.

Reply:
Are you thinking of zeitwerk? I can give that a try.

Reply:
was thinking of something far simpler like: 

https://github.com/ruby/ruby/blob/d92f09a5eea009fa28cd046e9d0eb698e3d94c5c/lib/bundler.rb#L42